src/olympia/shelves/admin.py

```diff
@@ -1,6 +1,45 @@
-from django.contrib import admin
+import requests
+
+from rest_framework.reverse import reverse as drf_reverse
+
+from django.contrib import admin, messages
+
+from .forms import ShelfForm
 
 
 class ShelfAdmin(admin.ModelAdmin):
     list_display = ('title', 'shelf_type')
     actions = ['delete_selected']
+    form = ShelfForm
+
+    def message_user(self, request, message, level=messages.INFO,
+                     extra_tags='', fail_silently=False):
+        pass
```

Comment:
what's this method for?  (it doesn't do anything)

Reply:
It overrides the automated success message after a shelf is created/updated.

Without it :
<img width="722" alt="Screen Shot 2020-07-27 at 4 33 35 PM" src="https://user-images.githubusercontent.com/28129806/88589435-1f8f2c00-d027-11ea-9e77-fc62ef59d4e1.png">

With it:
<img width="722" alt="Screen Shot 2020-07-27 at 4 34 11 PM" src="https://user-images.githubusercontent.com/28129806/88589513-44839f00-d027-11ea-84c3-852bae006c18.png">

Reply:
okay, can you add a code comment in the method saying that.